### PR TITLE
Project chatbot 

### PR DESCRIPTION
**Pull request by AnnikaSonnek**
See comments in ReadMe

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -15,14 +189,17 @@ const showMessage = (message, sender) => {
...
+        <img src="assets/anime.png" alt="User" />  
       </section>
     `
+    //Above, we get ahold of chat and add som innerHTML using += (add something to it). The show message function will be re-used several times. The innerHTML section will be added to the chat-section in the HTML-dokument. 
     // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
-  } else if (sender === 'bot') {
+  } 
+  else if (sender === 'bot') {
+    
     chat.innerHTML += `
```
Comment (AjmalZ) [inline code/script.js:199]: If you want to save time and set a delay on every bot/user message you could add the setTimeout to the else/if statements like this this:

```javascript
else if (sender === 'bot') {
setTimeout(() => {
    chat.innerHTML += `
      <section class="bot-msg">
        <img src="assets/bot.png" alt="Bot" />
        <img src="assets/kawaii.png" alt="Kawaii" />
        <div class="bubble bot-bubble">
          <p>${message}</p>
        </div>

  chat.scrollTop = chat.scrollHeight;
}, 1000);
}
```

File: code/script.js [hunk 2]
```diff
@@ -1,10 +1,184 @@
...
+
 
 // If you need any global variables that you can use across different functions, declare them here:
 
 
-// Declare your functions after this comment
+//**FUNCTION** Handle name input
+const handleNameInput = (event) => {
+  event.preventDefault(); //prevents the page from refreshing
+  //Store the value in a variable so we can access it after we clear it from the input
```
Comment (AjmalZ) [inline code/script.js:15]: Well explained, easy to follow!

**Review by AjmalZ — COMMENTED** [review]
The code in this project is well written and is easy to follow. The comments explains the code very well. The code fulfills all requirement for this project. The bot and website looks beautiful. Well done!

File: code/script.js (hunk 1 above)
Comment (AnnikaSonnek) [reply code/script.js:199]: Oh yeah, good idea! Thanks for the tip!

**Pull request closed without merging**